src/mocha_context.js

```diff
@@ -1,39 +0,0 @@
-/* eslint-env goodeggs/server-side-test */
-
-import assert from 'assert';
-import chai from 'chai';
-import sinon from 'sinon';
-
-beforeEach('set assert', function () {
-  this.assert = assert;
-});
-
-beforeEach('set expect', function () {
-  this.expect = chai.expect;
-});
-
-beforeEach('set up a sinon sandbox', function () {
-  this.sinon = sinon.sandbox.create({
-    injectInto: this,
-    properties: ['spy', 'stub', 'mock'],
-    useFakeTimers: false,
-    useFakeServer: false,
-  });
-
-  const clock = this.clock = this.sinon.useFakeTimers({
-    now: 0,
-    toFake: ['Date'],
-  });
-
-  this.stubTime = function stubTime (date) {
-    clock.setSystemTime(date);
-    return date;
-  };
-});
-
-afterEach('restore the sinon sandbox', function () {
-  this.clock.restore();
-  this.sinon.restore();
-});
-
-process.nextTick(() => delete require.cache[require.resolve(__filename)]); // eslint-disable-line
```

Comment:
did you end up verifying whether removing this breaks anything?

Reply:
Uh, tests pass in this module for two separate test files. Do you remember what wouldn't work? Was this a hack due to the magic side effects of importing the file?

Reply:
I have zero idea what this was for. Seems worth linking this branch into one of our apps and trying it though.

Reply:
I added this. It was an ugly ugly hack to get `mocha --watch` to work. I have no idea if it's still necessary. But please don't break `mocha --watch`!

Reply:
@demands ah ok. I experimented with a few test files in pick-api, and mocha watch works as expected with the beforeEach hooks.